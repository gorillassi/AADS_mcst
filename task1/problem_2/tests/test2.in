visit google.com
visit linux.org
visit youtube.com
back 1
visit facebook.com
back 1
forward 1