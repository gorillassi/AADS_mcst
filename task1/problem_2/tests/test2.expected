homepage
google.com
linux.org
youtube.com
linux.org
facebook.com
linux.org
facebook.com